db/models/news.model.js

```diff
@@ -85,3 +84,23 @@ exports.selectArticleWithCommentsByID = (article_id) => {
       throw error;
     });
 };
+
+exports.selectUserByUsername = (username) => {
+  return db.query(`SELECT * FROM users WHERE username = $1;`, [username])
+  .then((response) => {
+      return response.rows[0];
+  })
+  .catch((error) => {
+      throw error;
+  });
+};
+
+exports.insertComment = (article_id, username, body) => {
+    return db.query(`INSERT INTO comments (article_id, author, body) VALUES ($1, $2, $3) RETURNING *;`, [article_id, username, body])
+    .then((response) => {
+      return response.rows[0];
+    })
+    .catch((error) => {
+      throw error;
```

Comment:
unnecessary catch. as the catch block where your insertComment function is invoked (in the controller) will catch the error.